### PR TITLE
Allow cimpler to connect to a remote cimpler server

### PR DESCRIPTION
**Pull request by davidrans**
The host was defaulting to `localhost`. This allows the host to be set
via the config file or a CLI param.

### CODE REVIEW

File: bin/cimpler [hunk 1]
```diff
@@ -30,6 +34,11 @@ args           = require('optimist')
 
 var command = args._[0];
 
+connOptions = {
```
Comment (marczych) [inline bin/cimpler]: `var connOptions = ...`?

File: lib/http-trigger.js [hunk 2]
```diff
@@ -9,13 +9,16 @@ var http     = require('http'),
  * the same http port as the CLI plugin.
  */
 exports.injectBuild = function injectBuild(build, options, callback) {
-   httpPort = (options && options.httpPort)
+   var httpHost = (options && options.httpHost)
+               || loadHostFromConfig();
```
Comment (marczych) [inline lib/http-trigger.js]: No default host? `localhost` seems like a great default to me.

File: bin/cimpler (hunk 1 above)
Comment (davidrans) [inline bin/cimpler]: connectionOptions seemed too long, but I guess this is too hard to read

File: lib/http-trigger.js (hunk 2 above)
Comment (davidrans) [inline lib/http-trigger.js]: that's a good idea

File: bin/cimpler (hunk 1 above)
Comment (davidrans) [inline bin/cimpler]: e600588

File: lib/http-trigger.js (hunk 2 above)
Comment (davidrans) [inline lib/http-trigger.js]: af2a2f3

File: bin/cimpler (hunk 1 above)
Comment (marczych) [inline bin/cimpler]: I was actually pointing out that the `var` is missing so it's a global variable. I do like `connectionOptions` more than `connOptions`, though!

**Comment by marczych** [comment]
CR :melon: 

It would be nice to have passing tests, though.

**Comment by danielbeardsley** [comment]
To be fair, [master has the exact same failure](https://travis-ci.org/danielbeardsley/cimpler/branches).

I'm not sure why it is, I've tried many things but something about the environment of Travis is slightly different than where I've done development. Maybe bash versions, maybe some shell options, ...

**Pull request merged**